src/app/onion/learning-object-builder/components/outcome/outcome-typeahead/outcome-typeahead.component.ts

```diff
@@ -141,16 +141,15 @@ export class OutcomeTypeaheadComponent implements OnInit, OnChanges, OnDestroy {
    * @return {boolean} true if verb is found, false otherwise
    */
   isGoodVerb(verb: string, noEmit?: boolean): boolean {
-    const levelsArray = Array.from(levels.values());
-    for (let i = 0, l = levelsArray.length; i < l; i++) {
-      // for this level, grab it's verbs and check if the current verb is in that list
-      if (taxonomy.taxons[levelsArray[i]].verbs.includes(verb.toLowerCase())) {
-        this.bloom = levelsArray[i];
-        if (!noEmit) {
-          this.selectedCategory.emit(this.bloom);
-        }
-        return true;
+    if (
+      this.bloom &&
+      taxonomy.taxons[this.bloom] &&
+      taxonomy.taxons[this.bloom].verbs.includes(verb.toLowerCase())
+    ) {
+      if (!noEmit) {
+        this.selectedCategory.emit(this.bloom);
```

Comment:
If you copy an outcome into the text box can it still be recognized correctly? I noticed in the demo construct typed in was throwing an error.

Reply:
Actually thought that was normal behavior at first, but switched to main and see that yea it shouldn't do that. I'll make some edits @pzalep1